### DISCUSSION TITLE
coca ViT-L-14 training for classification

### DISCUSSION THREAD

**Discussion by wiamRB** [mlfoundations/open_clip#880] [Q&A]
I want to train coca ViT-L-14 on my own data set for classification, but I didn't find any tutorial for this.
If any one has an idea about it could you share instructions or code?!

**Comment by FatimaEljaimi** [comment]
> I want to train coca ViT-L-14 on my own data set for classification, but I didn't find any tutorial for this. If any one has an idea about it could you share instructions or code?!


I'm facing the same problem.

**Comment by Walid-AMARA** [comment]
I tried making some research on the matter, but I didn't find anything for a coca ViT-L-14 for only classification @wiamRB
